src/main/config/emissary.spi.ClassLocationVerificationProvider.cfg

```diff
@@ -0,0 +1,2 @@
+CLASS_NAME_emissary.analyze.Analyzer = "emissary"
+CLASS_NAME_emissary.analyze.Analyzer = "notexist"
```

Comment:
This works just like I anticipated, however we should take this 'fail' case out right?

Reply:
Yes, this should be removed. Left in by mistake after testing failure case.